src/main/java/org/openpnp/machine/reference/feeder/ReferenceDragFeeder.java

```diff
@@ -155,8 +157,6 @@ public void feed(Nozzle nozzle) throws Exception {
 	        }
 		}
 
-        head.moveToSafeZ();
```

Comment:
Why remove this? I believe you can't blindly assume safe Z when `feed()` is called.

Reply:
I will revisit after I get done with another PR.

Reply:
Reverted